### ISSUE TITLE
No detection from example images

### ISSUE THREAD

**Issue by ZilongZhang** [ultralytics/yolov5#5924]
Hi I am new to Yolo and I am trying to run the model on my computer, the model does not detect any object from the two example images (i.e. bus and zidane). I used models of different sizes but none of them succeeded. I wonder what went wrong and how to fix it. Here are some details about my environment:

  Windows 10
  Python 3.8
  Torch 1.10.0

Here are what Yolo prints when detecting objects from bus.jpg:

Using cache found in /.cache\torch\hub\ultralytics_yolov5_master
YOLOv5  2021-12-8 torch 1.10.0 CUDA:0 (NVIDIA GeForce GTX 1650, 4096MiB)
Fusing layers... 
Model Summary: 367 layers, 46533693 parameters, 0 gradients
Adding AutoShape... 
image 1/1: 1080x810 **(no detections)**
Speed: 17.0ms pre-process, 307.2ms inference, 0.0ms NMS per image at shape (1, 3, 640, 480)

**Comment by glenn-jocher** [comment]
@ZilongZhang this is probably due to CUDA 11 on your system and downgrading to CUDA 10 will fix this. 

There is an open issue ongoing with PyTorch and CUDA 11 unfortunately that is affecting multiple users when running AMP or FP16 inference. If this is the cause then detect.py and val.py will work but PyTorch Hub and train.py will not work for you under CUDA 11.

**Comment by glenn-jocher** [comment]
@ZilongZhang good news 😃! Your original issue may now be fixed ✅ in PR #5926. This PR default PyTorch Hub models to Automatic Mixed Precision (AMP) disabled status for improved compatibility with CUDA 11 under Windows and/or conda environments. AMP inference can now be toggled with the `model.amp` attribute:

```
import torch

# Model
model = torch.hub.load('ultralytics/yolov5', 'yolov5s')  # or yolov5m, yolov5l, yolov5x, custom
model.amp = False  # <--- disable AMP inference (solves Windows/conda/CUDA11 issues)
model.amp = True  # <--- enable AMP inference
```

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
